packages/runtime-dom/src/components/TransitionGroup.ts

```diff
@@ -112,7 +112,7 @@ const TransitionGroupImpl: ComponentOptions = {
         tag = 'span'
       }
 
-      prevChildren = children
+      prevChildren = children?.filter((c: VNode) => c.el?.getBoundingClientRect)
```

Comment:
```suggestion
      prevChildren = children && children.filter((c: VNode) => c.el?.getBoundingClientRect)
```

We don't use optional chaining here. (considering to bundle size)

Reply:
done